app/code/Magento/Customer/Model/Options.php

```diff
@@ -53,23 +57,43 @@ public function getNamePrefixOptions($store = null)
      */
     public function getNameSuffixOptions($store = null)
     {
-        return $this->_prepareNamePrefixSuffixOptions($this->addressHelper->getConfig('suffix_options', $store));
+        return $this->prepareNamePrefixSuffixOptions(
+            $this->addressHelper->getConfig('suffix_options', $store),
+            $this->addressHelper->getConfig('suffix_show', $store) == NooptreqSource::VALUE_OPTIONAL
+        );
+    }
+
+    /**
+     * @param $options
+     * @param bool $isOptional
+     * @return array|bool
+     *
+     * @deprecated See prepareNamePrefixSuffixOptions
```

Comment:
You should also be able to use http://docs.phpdoc.org/references/phpdoc/tags/see.html here to link to the new method. Something like the following should work.

```
@see prepareNamePrefixSuffixOptions()
```

Reply:
Changed, makes sense.